docs/src/modules/java/pages/replicated-entity.adoc

```diff
@@ -0,0 +1,192 @@
+
+By default, the name of the command that the method handles will be the name of the method with the first letter capitalized. So, a method called `getCart` will handle a gRPC service call command named `GetCart`. This can be overridden by setting the `name` parameter on the `@CommandHandler` annotation.
+
+The command handler also can take the gRPC service call input type as a parameter to receive the command message. This is optional, sometimes it's not needed. For example, our `GetCart` service call doesn't need any information from the message, since it's just returning the current state as is. Meanwhile, the `AddItem` service call does need information from the message, since it needs to know the product id, description and quantity to add to the cart.
+
+The return type of the command handler must be the output type for the gRPC service call, this will be sent as the reply.
+
+The following shows the implementation of the `GetCart` command handler. This command handler is a read-only command handler, it doesn't update the Replicated Entity, it just returns some state:
+
+[source,java,indent=0]
+----
+include::example$replicatedentity/ShoppingCartEntity.java[tag=get-cart]
+----
+
+== Updating a Replicated Entity
+
+Due to Akka Serverless's take-in-turns approach, Replicated Entities may only be updated in command handlers and <<Responding to stream cancellation,stream cancellation callbacks>>.
+
+Here's a command handler for the `AddItem` command that adds the item to the shopping cart:
+
+[source,java,indent=0]
+----
+include::example$replicatedentity/ShoppingCartEntity.java[tag=add-item]
+----
+
+== Deleting a Replicated Entity
+
+A Replicated Entity can be deleted by invoking link:{attachmentsdir}/api/com/akkaserverless/javasdk/replicatedentity/CommandContext.html#delete--[`CommandContext.delete`{tab-icon}, window="new"]. Once deleted, the Entity will be shut down, and all subsequent commands for the Entity will be rejected.
+
+Caution should be taken when deleting Replicated Entities--Akka Serverless needs to maintain tombstones for each deleted Replicated Entity. Over time, if many Replicated Entities are created and deleted, this will result in not just running out of memory, but increased network usage as the tombstones still need to be gossipped through the cluster for replication.
+
+== Streamed command handlers
+
+Replicated Entities support streaming over gRPC. Streamed commands can be used to receive and publish updates to the state. If a gRPC service call has a streamed result type, the handler for that call can accept a link:{attachmentsdir}/api/com/akkaserverless/javasdk/replicatedentity/StreamedCommandContext.html[`StreamedCommandContext`{tab-icon}, window="new"], and use that to register callbacks.
+
+
+
+=== Responding to changes
```

Comment:
this has been removed

Reply:
Yes, the whole `Streamed command handlers` section should be removed.